products/fluent-bit.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Fast and Lightweight Log processor and forwarder
+category: app
+tags: kubernetes
+permalink: /fluent-bit
+alternate_urls:
+-   /fluentbit
+versionCommand: |-
+  fluent-bit -v
+releasePolicyLink: https://fluentbit.io/announcements/older-versions
+changelogTemplate: https://github.com/fluent/fluent-bit/releases/tag/v__LATEST__
+releaseDateColumn: true
+eolColumn: Support
+
+auto:
+  methods:
+  -   git: https://github.com/fluent/fluent-bit
+
+identifiers:
+-   purl: pkg:github/fluent/fluent-bit
+-   purl: pkg:docker/fluent/fluent-bit
```

Comment:
This will add a link to the various distro packages: https://repology.org/project/fluent-bit/versions

```suggestion
-   purl: pkg:github/fluent/fluent-bit
-   purl: pkg:docker/fluent/fluent-bit
-   repology: fluent-bit
```